### ISSUE TITLE
Schedule for moving onnx optimizers to this new repo

### ISSUE THREAD

**Issue by askhade** [onnx/optimizer#1]
@daquexian @fumihwh : Could you add details regarding the planned schedule for moving optimizers to this repo along with ETAs. Thanks!

**Comment by daquexian** [comment]
@askhade Thanks for reminding me!

As discussed in infra SIG meeting (https://github.com/onnx/sigs/blob/master/infra/meetings/002-20200806.md), optimizer code in [onnx/onnx](https://github.com/onnx/onnx) repo will be moved to this repo and maintained actively by the community. After discussing with @fumihwh, here is the planned timeline:

| Date  | Content |
| ------------- | ------------- |
| 08.31  | Copy optimizer code from onnx/onnx repo to onnx/optimizer repo, ensure the same functionality. <br/> Build an "onnx-optimizer" (or another better name) python package. <br/> Add warning message in onnx/onnx repo to encourage users to use the new repo. |
| 10.10  | Fix critical bugs: <br/> errors when IR version >= 4 <br/> fuse_bn_into_conv breaks the graph or introduces wrong parameters  |
| 1.8 Release |  |
| After 1.8 Release   | New feature development, including (still need some discussion): <br/> New user-friendly API <br/> New IR for optimizers. <br/> Replace tensor.h with a mature tensor library like xtensor <br/> Integrate with ONNX Function operators (e.g., unfold function ops and do some optimization) <br/> Pattern matching like tensorflow-onnx <br/> 
| Before 1.9 Release |  remove optimizer codes in ONNX repo

**Comment by askhade** [comment]
@daquexian : Add warning message in onnx/onnx repo to advise users to use the new package. When will this be done?

08.31 | Copy optimizer code from onnx/onnx repo to onnx/optimizer repo, ensuring the same functionality.Build an "onnx-optimizer" (or another better name) python package. Add warning message in onnx/onnx repo to advise users to use the new package.
-- | --


I think the optimizer package creation should be done only after fixing all the pending issues specially the IR gap. I do not see a point in publishing a package when the underlying code is not really useful because of bugs and incomplete support.

**Comment by daquexian** [comment]
> I think the optimizer package creation should be done only after fixing all the pending issues specially the IR gap. I do not see a point in publishing a package when the underlying code is not really useful because of bugs and incomplete support.

I don't think so. This repo has at least the same functionality with the optimizer in onnx/onnx repo. We should encourage users to use this repo so that they can benefit from the coming bugfixes (like #4).

I have published a package named onnxoptimizer on pypi yesterday (I had to wait for the merging of #2 so delayed a bit), and will submit a PR to add warning message in onnx/onnx repo.
